sky/cli.py

```diff
@@ -1750,17 +1761,23 @@ def status(all: bool, refresh: bool, ip: bool, show_spot_jobs: bool,
                           show_all=False,
                           limit_num_jobs_to_show=not all,
                           is_called_by_user=False))
-        if ip:
+        if ip or ports:
```

Comment:
To discuss: Probably we could have an enum containing `ip` and `ports`, and use sth like `--target ip`/`--target ports` in the CLI?